src/InputArea.vue

```diff
@@ -0,0 +1,207 @@
+<template>
+  <div class="wrapper">
+    <textarea
+      ref="textarea"
+      :value="value"
+      rows="2"
+      cols="30"
+      class="form-control"
+      :disabled="disabled"
+      @input="onInput"
+    />
+    <emoji-picker :search="search" @emoji="insertEmoji">
+      <button
+        slot="emoji-invoker"
```

Comment:
It looks like you're using deprecated syntax here https://v2.vuejs.org/v2/guide/components-slots.html#Deprecated-Syntax

Reply:
Fixed